subschedule.go

```diff
@@ -27,6 +27,16 @@ const (
 	SubscriptionScheduleStatusTrialing  SubscriptionScheduleStatus = "released"
 )
 
+// SubscriptionScheduleBillingCycleAnchor is the list of allowed values for the
+// schedule's billing_cycle_anchor.
+type SubscriptionScheduleBillingCycleAnchor string
```

Comment:
I'm curious if this is the right name? That naming implies "top level" and it's not. It's nested in DefaultSettings and Phases. I think we want `SubscriptionSchedulePhaseBillingCycleAnchor` instead

Reply:
Okay, yeah I can add that name wasn't sure if we add Phase if that implies it wouldn't be used for default settings, but I can see that a default settings is kinda like an abstract version of a phase?

Reply:
yep that's how my brain thought of this and how I usually name things. No idea how we will encode that in codegen lol